From: Wu Fengguang <fengguang.wu@intel.com>
Date: Wed, 8 Feb 2017 03:42:05 +0800
Subject: net: qcom/emac: fix semicolon.cocci warnings
Patch-mainline: v4.11-rc1
Git-commit: 3c19bd6c52d441893ba19b3418825b27cfa4fd9c
References: FATE#320512

drivers/net/ethernet/qualcomm/emac/emac-ethtool.c:155:49-50: Unneeded semicolon

 Remove unneeded semicolon.

Generated by: scripts/coccinelle/misc/semicolon.cocci

CC: Timur Tabi <timur@codeaurora.org>
Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/net/ethernet/qualcomm/emac/emac-ethtool.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/qualcomm/emac/emac-ethtool.c
+++ b/drivers/net/ethernet/qualcomm/emac/emac-ethtool.c
@@ -152,7 +152,7 @@ static void emac_get_pauseparam(struct n
 
 	pause->autoneg = adpt->automatic ? AUTONEG_ENABLE : AUTONEG_DISABLE;
 	pause->rx_pause = adpt->rx_flow_control ? 1 : 0;
-	pause->tx_pause = adpt->tx_flow_control ? 1 : 0;;
+	pause->tx_pause = adpt->tx_flow_control ? 1 : 0;
 }
 
 static int emac_set_pauseparam(struct net_device *netdev,
